docs/source/features.rst

```diff
@@ -1,11 +1,15 @@
+.. _Features:
+
 Features
 ********
+Fab is an open source Python project. Feel free to get involved.
```

Comment:
Yes please!

Reply:
It turns out it was more than a paragraph. :-)

> Why a New Build Tool?
>
> There are several existing build tools, such as Make, and many more build system
> builders such as CMake out there. So why muddy these already turgid waters?
>
> A number of popular solutions to the problem of building software were examined and
> it became apparent that there was a fundamental mismatch in requirements.
>
> Most build systems are designed from the position that the executable is the product. The
> source files needed to create that executable are listed because they change infrequently
> and  are merely a means to the end of creating an executable.
>
> Scientific software can be rather different. In this paradigm the source is the product and
> the executable merely a handy means to access it's power. When building is viewed from
> this point-of-view maintaining lists of source is restricting because it changes so often.
>
> In this light it makes sense to develop a new build tool which gives primacy to the source
> rather than the executable. The ultimate goal is to support science developers in their
> work.

Reply:
That's now in at the top of the main page, thanks.

Reply:
It is indeed!